specification/reservations/resource-manager/Microsoft.Capacity/preview/2019-04-01/reservations.json

```
@@ -27,6 +27,50 @@
     }
   },
   "paths": {
+    "/providers/Microsoft.Capacity/reservationOrders/{reservationOrderId}/reservations/{reservationId}/availableScopes": {
+      "post": {
+        "summary": "Get Available Scopes for `Reservation`.",
+        "description": "Get Available Scopes for `Reservation`.\n",
+        "operationId": "Reservation_AvailableScopes", 
+		"x-ms-examples": {
+          "AvailableScopes": {
+            "$ref": "./examples/GetAvailableScope.json"
+          }
+        },		
+        "x-ms-long-running-operation": true,
+        "parameters": [
+		  {
+            "$ref": "#/parameters/ReservationOrderIdParameter"
+          },
+          {
+            "$ref": "#/parameters/ReservationIdParameter"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/ScopeParameter"
+          }
+        ],
+        "tags": [
+          "Reservation, AvailableScopes"
```

Note sure if these are supposed to be separate tags
```suggestion
          "Reservation", "AvailableScopes"
```

---

Not sure, but I checked the format of our previous swagger. Same as what I put.

---

Let me know if I need to change this format.

---

Sorry, one of the SDK/ARM folks would need to make that call. I just noticed that it looked a little odd